cibuildwheel/bashlex_eval.py

```diff
@@ -65,9 +74,36 @@ def evaluate_word_node(node: bashlex.ast.node, context: NodeExecutionContext) ->
 
 
 def evaluate_command_node(node: bashlex.ast.node, context: NodeExecutionContext) -> str:
-    words = [evaluate_node(part, context=context) for part in node.parts]
```

Comment:
What happened before, then? `';'` was just not supported either?

Reply:
Not within command substitutions, no.